GettyImages.Api/WebHelper.cs

```diff
@@ -266,36 +332,6 @@ internal WebHelper(Credentials credentials, string baseAddress, DelegatingHandle
         return mainHandler;
     }
 
-    private static async Task HandleResponseAsync(HttpResponseMessage httpResponse)
```

Comment:
I'm happy to see code being refactored out of the `WebHelper` class. I'd like to see us refactor this entire class away at some point.